pkg/providers/amifamily/suite_test.go

```diff
@@ -553,6 +729,72 @@ var _ = Describe("AMIProvider", func() {
 				},
 			))
 		})
+		It("should sort deprecated amis with the same name and deprecation time consistently", func() {
```

Comment:
While I don't think this will likely _ever_ be hit, should we order by ID on name ties just to be _absolutely_ sure that there's no collisions? This only preserves the order of the input to the function, where the input isn't guaranteed to be in the same order.

Reply:
nvm, we already do!